build.gradle

```diff
@@ -10,7 +10,7 @@ repositories {
 	mavenCentral()
 }
 
-def runeLiteVersion = '1.8.9'
+def runeLiteVersion = 'latest.release'
```

Comment:
This isn't really relevant to this PR, but I think it saves a step for future contributors